### ISSUE TITLE
Exit the server itself after migration completes

### ISSUE THREAD

**Issue by pamodaaw** [wso2/product-is#4587]
#### Description:
Automate migration client to terminate the running server after successfully completing the migration. 

#### Suggested Labels:
Improvement 

#### Affected Product:
Migration Tool

**Comment by wojwoj** [comment]
Hi,
It would be really nice if the sever could be started only for the migration and close itself after migration is done.
From my point of view, if server is started to perform only migration it should close itself always regardless of migration result. The description of this issue says "after successfully completing the migration", but I would like the sever to stop also when the migration process fails.
Moreover, the server could return result code: 0 if migration was successful and 1 if it fails.

To sum up, I would suggest to small improvements:

1. If server is started to perform only migration, it should stop once the migration is completed regardless of migration result
2. The script used to start the server to perform migration should return result code: 0 for successful migration and 1 for failed migration

Kind Regards,
Wojtek

**Comment by Krzysztof1985** [comment]
Hello, 

I totally agree with @wojwoj . If somebody would like to perform just migration, then the flow should be look like start the server (just for migration)
shut down the server with some summary.

**Comment by isharak** [comment]
This issue is being closed due to extended inactivity. Please feel free to reopen it if further attention is needed. Thank you for helping us keep the issue list relevant and focused!
